apollo/README.md

```diff
@@ -20,31 +20,33 @@ The Datadog metrics forwarded by Graph Manager are:
+These metrics are aggregated in 60-second intervals and tagged with the GraphQL operation name as `operation:<query-name>`. Unique query signatures with the same operation name are merged, and queries without an operation name are ignored.
 
-All of the metrics are also tagged with the Graph Manager graph ID as `service:<GRAPH_ID>` and the variant name as `variant:<VARIANT_NAME>`, so multiple graphs from Graph Manager can send data to the same Datadog account. If you have not set a variant name, then "current" will be used.
-
-If you're reporting metrics to Graph Manager through the Engine proxy, Datadog will merge your statistics across multiple instances of the proxy (per-host metrics are not available). Just like in the Graph Manager UI, each operation inside a query batch is counted individually.
+These metrics are also tagged with both the associated Studio graph ID (as `service:<graph-id>`) and the associated variant name (as `variant:<variant-name>`), so multiple graphs from Studio can send data to the same Datadog account. If you haven't set a variant name, then `current` is used.
 
 ## Setup
 
 ### Configuration
 
-Getting set up with the Apollo Datadog integration is as simple as providing a Datadog API key to Graph Manager. There's no further configuration required.
+Getting set up with the Apollo Datadog integration is as simple as providing a Datadog API key and region to Studio. There's no further configuration required.
+
+1. Go to your [Datadog integrations page][2] and click on the Apollo tile. Then go to the **Configuration** tab and click **Install Integration** at the bottom.
 
-1. Go to the [Datadog integrations page][2] and click on the Apollo tile. Go to the **Configuration** tab, scroll to the bottom, and press **Install Integration**.
+2. Go to your [Datadog APIs page][3] and create an API key.
 
-2. Go to the [Datadog APIs page][3] and create an API key.
+3. Determine your Datadog API region by looking at your browser's address bar:
+- If the domain name is `app.datadoghq.com`, then your API region is `US`.
+- If the domain name is `app.datadoghq.eu`, then your API region is `EU`.
 
-3. In [Graph Manager][4], go to the integrations page for your graph.
+4. In [Studio][4], go to your graph's Integrations page:
 
    ![IntegrationsPage][5]
 
-4. Toggle the Datadog integration to turn it on. Paste the API key, and press **Save**. You can use the same API key for all your graphs, since all metrics are tagged with the graph ID (`service:<GRAPH_ID>`).
+5. In the Datadog Forwarding section, click **Configure**. Provide your API key and region, then click **Enable**. You can use the same API key for all of your graphs, because all forwarded metrics are tagged with the corresponding graph's ID (`service:<graph-id>`).
```

Comment:
```suggestion
5. In the Datadog Forwarding section, click **Configure**. Provide your API key and region, then click **Enable**. Because all forwarded metrics are tagged with the corresponding graph's ID (`service:<graph-id>`), you can use the same API key for all of your graphs.
```